modules/system/reportwidgets/Status.php

```diff
@@ -127,8 +127,8 @@ protected function getSystemWarnings()
             }
         }
 
-        foreach ($missingPlugins as $pluginCode) {
-            $warnings[] = Lang::get('backend::lang.warnings.plugin_missing', ['name' => '<strong>'.$pluginCode.'</strong>']);
+        foreach ($missingPlugins as $plugin) {
+            $warnings[] = Lang::get('backend::lang.warnings.plugin_missing', ['name' => '<strong>'.$plugin['code'].'</strong>']);
```

Comment:
This is using a different lang key, we should switch it to using the new key and remove the old key if it is no longer used.

Reply:
Do we have to? It is already translated on every language

Reply:
Fair enough, it'll take them to the updates page where they'll get more information so I'm fine leaving it as is for now.